### PR TITLE
feat(fields): provide fields component

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

File: framework/atoms/fields/fields.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <div class="field__items">
+    <div class="field__item field__item--expired">
+      12 October 2016 (CET)
+    </div>
+  </div>
+</div>
+<div class="field social-media-links--field field-label-inline clearfix">
+  <div class="field__label">Social media</div>
+  <ul class="field__items social-media-links__list social-media-links__horizontal">
+    <li class="social-media-links__item icon-social--smaller">
```
Comment (kalinchernev) [inline framework/atoms/fields/fields.html]: tricky, dependency to something we don't have + icons

File: framework/atoms/fields/fields.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <a href="fb.com" title="Facebook" class="icon-social icon-social--facebook">Facebook</a>
+    </li>
+    <li class="social-media-links__item icon-social--smaller">
+      <a href="goo.gl" title="G+" class="icon-social icon-social--gplus">G+</a>
```
Comment (kalinchernev) [inline framework/atoms/fields/fields.html]: this one has to be handled for sure

File: framework/atoms/fields/fields.html (hunk 1 above)
Comment (yhuard) [inline framework/atoms/fields/fields.html]: Yup, good catch. I'll implement "social media links" and then I'll come back to this one

Comment (yhuard) [inline framework/atoms/fields/fields.html]: PR ready: https://github.com/ec-europa/europa-component-library/pull/25

**Pull request merged**